modules/Crowd Assessment/crowdAssess_view_discuss_postProcess.php

```diff
@@ -123,16 +123,22 @@
 
                         $homeworkNameSingular = $container->get(SettingGateway::class)->getSettingByScope('Planner', 'homeworkNameSingular');
 
+                        $notificationGateway = new \Gibbon\Domain\System\NotificationGateway($pdo);
```

Comment:
Could we add Gibbon\Domain\System\NotificationGateway and Gibbon\Comms\NotificationSender as `use` declarations at the top, rather than putting the full namespace in the code.

Reply:
This makes a lot of sense in terms of ease of changing namespaces. I've just made the changes and will commit after I sort out the DRY-related comments below.